### DISCUSSION TITLE
Port to Deno

### DISCUSSION THREAD

**Discussion by clayrisser** [loopbackio/loopback-next#7199] [Ideas]
I think it would be great if Loopback 4 could be ported to Deno.

https://deno.land

**Comment by deepakrkris** [comment]
@codejamninja could you please elaborate more on what are the advantages of supporting deno ? also would you be able to contribute or create a prototype ?

**Reply by ganjarsetia** [reply]
One of the advantage is, it can compile to executable file (standalone)
